spyder_terminal/widgets/main_widget.py

```diff
@@ -355,7 +355,10 @@ def create_new_term(self, name=None, give_focus=True, path=None):
         """Add a new terminal tab."""
         if path is None:
             path = self.current_cwd
-        path = path.replace('\\', '/')
+        if self.project_path is not None:
+            path = self.project_path
+        else:
+            path = path.replace('\\', '/')
```

Comment:
```suggestion
        path = path.replace('\\', '/')
```

I think this replacement should be applied after the cwd or the project path are selected as `path`. At least that was the case before.